### DISCUSSION TITLE
如何卸载nonebot。

### DISCUSSION THREAD

**Discussion by Spencer-star** [nonebot/discussions#91] [问答]
在使用pip uninstall nonebot2。提示WARNING: Skipping nonebot as it is not installed.

是通过nb-cli进行安装的b1版本。

**Comment by yanyongyu** [comment]
你确定你输入的是nonebot2而不是nonebot？

**Comment by buynonsense** [comment]
如何只卸载nb的某一个依赖？

**Reply by yanyongyu** [reply]
你要卸载什么。。。

**Reply by buynonsense** [reply]
就是我通过nb adapter install …… 正常安装了一个依赖后，我暴力迁移项目导致路径对不上。现在我想卸载这个依赖重新安装来解决这个问题应该如何做。
